### PR TITLE
dbeaver/pro#1735 Update tips

### PR DESCRIPTION
**Pull request by E1izabeth**
The existing tips seems to be rewritten in some places. I took the new full list from the issue.

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ext.ui.tipoftheday/tips.xml [hunk 1]
```diff
@@ -1,95 +1,231 @@
...
-	<tip>To execute the selected SQL expression and print the results use the following shortcut: <b>Ctrl+Alt+'</b>.</tip>
+	<!--        Tips below were reviewed on 30th of June 2023   -->
+	<tip>DBeaver can be customized to suit your needs. Learn detailed information about optional extensions <a href="https://dbeaver.com/docs/wiki/Optional-extensions"> here</a>.</tip>
+	<tip>You can <a href="prefs://org.eclipse.ui.window.customizePerspective">customize the menu bar and the list of menu items displayed</a> in the <b>Menu Visibility</b> tab.</tip>
+	<tip>You can <a href="prefs://org.eclipse.ui.window.customizePerspective">customize the toolbar </a> in the <b>Toolbar Visibility</b> tab.</tip>
+	<tip>You can create a keyboard shortcut for most commands in DBeaver. <a href="prefs://org.eclipse.ui.preferencePages.Keys">Existing shortcuts can also be changed</a></tip>
+	<tip>You can change the current <a href="https://dbeaver.com/2022/05/26/change-the-data-in-the-smart-way/">database transaction mode</a> in the toolbar.
+		 Click the dropdown menu to the right of the <b>Commit and Rollback buttons</b>.</tip>
```
Comment (Destrolaric) [inline plugins/org.jkiss.dbeaver.ext.ui.tipoftheday/tips.xml:10]: Could you look if there are any cases when wiki has more recent snapshot?

Comment (E1izabeth) [reply plugins/org.jkiss.dbeaver.ext.ui.tipoftheday/tips.xml:10]: It's not a wiki. It an article in the blog. There is only one version of it.

**Comment by MashaKorax** [comment]
Please, improve tips for macOs according to https://docs.google.com/document/d/1HCDj2cYFq1tYkaWgiOKIGx0PiUz9yQhOduGgjFxcglM/edit#heading=h.l96nti7f3pw7
Снимок экрана 2023-07-14 в 10 23 41
Снимок экрана 2023-07-14 в 10 24 12

**Comment by E1izabeth** [comment]
It can be done only after https://github.com/dbeaver/pro/issues/1781

**Comment by MashaKorax** [comment]
verified

**Pull request merged**